frontend/src/js/filters/filters.js

```diff
@@ -15,4 +15,21 @@
         };
     }
 
+    angular.module('evalai')
+            .filter('format_execution_time', format_execution_time);
+
+    function format_execution_time() {
+        return function(execution_time) {
+            var executiontime = new Date(execution_time * 1000);
+            var days = executiontime.getUTCDate() - 1;
+            var hours = executiontime.getUTCHours();
+            var minutes = executiontime.getUTCMinutes();
+            var seconds = executiontime.getSeconds();
+            var timeString = days.toString().padStart(2, '0') + ' day ' + hours.toString().padStart(2, '0') + ' hr ' + 
```

Comment:
@Alabhya268 let's not show `day` `'hr` or `min` when they have 0 value. Rest of the changes look fine.

cc: @RishabhJain2018